src/main/java/com/gooddata/GoodDataSettings.java

```diff
@@ -22,6 +24,7 @@
     private int connectionTimeout = secondsToMillis(10);
     private int connectionRequestTimeout = secondsToMillis(10);
     private int socketTimeout = secondsToMillis(60);
+    private String userAgent = StringUtils.EMPTY;
```

Comment:
Is this 3rd party constant "StringUtils.EMPTY" necessary? Why not to use more concise ""? Or why not to use default value (null)?